spring-data-jdbc/src/main/java/org/springframework/data/jdbc/repository/config/AbstractJdbcConfiguration.java

```diff
@@ -76,17 +79,28 @@ public JdbcMappingContext jdbcMappingContext(Optional<NamingStrategy> namingStra
 	 */
 	@Bean
 	public JdbcConverter jdbcConverter(RelationalMappingContext mappingContext, NamedParameterJdbcOperations operations,
-			@Lazy RelationResolver relationResolver) {
+			ObjectProvider<RelationResolver> relationResolverProvider, Optional<NamingStrategy> namingStrategy,
+			JdbcConverter jdbcConverter) {
+
+		RelationResolver relationResolver = relationResolverProvider.getIfAvailable(() -> dataAccessStrategy( //
```

Comment:
Can we try to avoid these manual breaks in parameters lists? If a list exceeds the line length, we could rather prefer breaking right before the invocation instead:

```
relationResolverProvider //
  .getIfAvailable(() -> …)
```